crates/sui-benchmark/tests/simtest.rs

```diff
@@ -547,7 +552,64 @@ mod test {
             info!("Simulated load config: {:?}", simulated_load_config);
         }
 
-        test_simulated_load_with_test_config(test_cluster, 50, simulated_load_config).await;
+        test_simulated_load_with_test_config(test_cluster, 50, simulated_load_config, None, None)
+            .await;
+    }
+
+    // Tests cluster defense against failing transaction floods Traffic Control
+    #[sim_test(config = "test_config()")]
+    async fn test_simulated_load_expected_failure_traffic_control() {
+        // TODO: can we get away with significatly increasing this?
```

Comment:
Yeah it's not a big deal that we do. PTN would be the real test. That said, in my experience, getting the tally timing to work correctly in simtests doesn't seem to work, I think owing to how the network is managed in a simtest. I don't recall the details tbh, but what I can say is that I've run this test with both low and high tps and a threshold that should trigger blocking, and what I see through logging is that we are tallying the error correctly but not blocking.

I think it may not be worth trying to figure out here though. The intent with the simtest is really just to ensure that this benchmark tooling doesn't break with these changes.

I'll remove the TODO comment here though